data-raw/weather.R

```diff
@@ -59,8 +59,11 @@ weather <- raw %>%
     wind_speed = as.numeric(wind_speed) * 1.15078, # convert to mpg
     wind_gust = as.numeric(wind_speed) * 1.15078
   ) %>%
-  mutate(year = 2013, month = month(time), day = mday(time), hour = hour(time)) %>%
+  mutate(year = 2013, month = month(time), day = mday(time),
+         hour = hour(time), minute = minute(time)) %>%
   group_by(station, month, day, hour) %>%
+  filter(minute <= 51) %>%
```

Comment:
Could you please insert a comment here briefly explaining (with link) why we filter this out, and use `max()` as a summary?